pages/index.js

```diff
@@ -1,83 +1,65 @@
-    bullet_best: bullet_best_stats["rating"],
-    daily_best:daily_best_stats["rating"]
-  }
-
-  document.querySelector("#avatar_image").src = dictionary.photo
-  document.querySelector("#avatar_name").innerHTML = "Name :" + dictionary.name
-  document.querySelector("#avatar_username").innerHTML = "Username :" +dictionary.username
-  document.querySelector("#avatar_league").innerHTML = "League :" + dictionary.league
-  document.querySelector("#avatar_rapid").innerHTML = "Best score in Rapid :"  + dictionary.rapid_best
-  document.querySelector("#avatar_bullet").innerHTML = "Best score in Bullet :" +dictionary.bullet_best
-  document.querySelector("#avatar_blitz").innerHTML = "Best score in Blitz :" + dictionary.blitz_best
-  document.querySelector("#avatar_daily").innerHTML = "Best score in Daily :" + dictionary.daily_best
-  document.querySelector("#avatar_location").innerHTML = "Location :" + dictionary.location
-
-
-  console.log(dictionary)
-
-  }catch{
-    console.log("runtime error")
-  }
-
-}
-
-function Chess(){
+import { useState } from "react";
+
+function Chess() {
+  const [userName, setUserName] = useState("");
+  const [playerDetails, setPlayerDetails] = useState();
+  const [playerStats, setPlayerStats] = useState();
+
+  const get_stats = async () => {
+    try {
+      const player_details_url = `https://api.chess.com/pub/player/${userName}`;
+      const player_details_response = await fetch(player_details_url);
+      const player_details = await player_details_response.json();
+
+      const player_stats_url = `https://api.chess.com/pub/player/${userName}/stats`;
+      const player_stats_response = await fetch(player_stats_url);
+      const player_stats = await player_stats_response.json();
+
+      setPlayerDetails(player_details);
+      setPlayerStats(player_stats);
+    } catch (error) {
+      console.log(error);
+      alert(error.message);
+    }
+  };
 
   return (
-      <div>
-          <label>Name</label>
-          <input type="text" name="name" id="username" onChange={get_stats}/> <br></br>
-
-          <img src="" id="avatar_image"></img>  <br></br>
-          <p1 id="avatar_name"></p1> <br></br>
-          <p1 id="avatar_username"></p1> <br></br>
-          <p1 id="avatar_location"></p1> <br></br>
-          <p1 id="avatar_league"></p1> <br></br>
-          <p1 id="avatar_rapid"></p1> <br></br>
-          <p1 id="avatar_blitz"></p1> <br></br>
-          <p1 id="avatar_bullet"></p1> <br></br>
-          <p1 id="avatar_daily"></p1> <br></br>
-
-
-          
-
-      </div>
-  )
+    <div className="center">
+      <label>Name</label>
+      <input
+        type="text"
+        name="name"
+        id="username"
+        value={userName}
+        onChange={(e) => {
+          setUserName(e.target.value);
+        }}
+      />
+      <button onClick={get_stats}>Get Stats</button>
+      <br></br>
+      {playerDetails && playerStats ? (
+        <>
+          <img src={playerDetails.avatar}></img>
```

Comment:
used Javascript in JSX with curly braces
For refrence see https://react.dev/learn/javascript-in-jsx-with-curly-braces